frontend/views/pages/GroupDashboard.vue

```diff
@@ -4,12 +4,12 @@ page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='groupSettings
 
   start-inviting(v-if='groupMembersCount === 1')
 
+  page-section(:title='Object.keys(currentGroupState.proposals).length > 0 ? L("Proposals") : ""')
```

Comment:
Having a page section with an empty title? That doesn't seem to make much sense?

Reply:
it makes sense following the designs.